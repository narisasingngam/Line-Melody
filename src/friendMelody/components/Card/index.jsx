import styled from "styled-components";

const Card = styled.div`
  width: fit-content;
  padding: 24px;
  margin: 20px;
  border-radius: 8px;
  background-color: white;
  cursor: pointer;
  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
  transition: 0.3s;
  &:hover {
    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
  }

  @media only screen and (max-width: 600px)  {
    width: 100px;
  }
`;

export default Card;