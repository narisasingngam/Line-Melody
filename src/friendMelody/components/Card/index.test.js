import React from "react";
import {render} from '@testing-library/react'
import Card from ".";
import 'jest-styled-components'

describe("Card", () => {
  it("renders Card correctly", () => {
    const {container} = render(<Card/>)

    expect(container.firstChild).toMatchSnapshot()
  });
});