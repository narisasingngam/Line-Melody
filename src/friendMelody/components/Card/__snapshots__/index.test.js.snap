// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders Card correctly 1`] = `
.c0 {
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  padding: 24px;
  margin: 20px;
  border-radius: 8px;
  background-color: white;
  cursor: pointer;
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  -webkit-transition: 0.3s;
  transition: 0.3s;
}

.c0:hover {
  box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
}

@media only screen and (max-width:600px) {
  .c0 {
    width: 100px;
  }
}

<div
  class="c0"
/>
`;
